projects/testing-library/types.d.ts

```diff
@@ -0,0 +1 @@
+declare function afterEach(fn: () => {}, timeout?: number): void;
```

Comment:
Is this needed to run tests in CI?

Reply:
In the IDE it shows an error (duplicate identifier)
![image](https://user-images.githubusercontent.com/7110786/75545229-08bbbd80-5a26-11ea-8431-6a530f369e85.png)

Reply:
Needed to be able to build